_posts/2017-04-30-raku.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: "Raku"
+code:
+  - "raku -e 'say 0.1+0.2'"
+  - "raku -e 'say (0.1+0.2).base(10, 17)'"
+  - "raku -e 'say 1/10+2/10'"
+  - "raku -e 'say (0.1.Num + 0.2.Num).base(10, 17)'"
```

Comment:
I would also suggest improving the formatting here.

The bug alluded to that requires calling `base` has long been resolved, and also `Num`'s can be forced more succinctly by using scientific notation.

The end result is that I'd prefer these 4 lines to look like this: -

  - "raku -e 'say 0.1 + 0.2'"
  - "raku -e 'say (0.1 + 0.2).fmt('%.17f')'"
  - "raku -e 'say 1/10 + 2/10'"
  - "raku -e 'say 0.1e0 + 0.2e0'"

The results of these lines remains the same.

Reply:
Good changes, except the single quotes inside single quotes :-)